.gitmodules

```diff
@@ -2,3 +2,6 @@
 	path = third_party/cutlass
 	url = https://github.com/NVIDIA/cutlass.git
 	branch = v2.4.0
+[submodule "third_party/cuda-samples"]
+	path = third_party/cuda-samples
+	url = https://github.com/NVIDIA/cuda-samples.git
```

Comment:
Do we need to add branch? Although it's useless actually.

Reply:
Besides, why not use [CUDA Samples v11.4](https://github.com/NVIDIA/cuda-samples/releases/tag/v11.4), the commit id is e950012e

Reply:
can not add branch it .gitmodules, due to it's not a real branch but only a tag. Besides,  CUDA Samples version is not necessary to match CUDA version in docker?

Reply:
Could we determine the version when build docker? For example, when build docker with cuda11.4 or cuda12 in the future, it can pull the corresponding version of cuda-samples?

We can checkout to the version before the building.

Reply:
could achieve this in `Makefile` (not tested):
```make
bandwidthTest:
    cd ./cuda-samples
    git checkout v$$(nvcc --version | grep 'release' | awk '{print $6}' | cut -c2- | cut -d '.' -f1-2)
    make -C Samples/bandwidthTest ...
```

Reply:
have added accordingly logic in makefile and test on A100 and V100, please check it

Reply:
Therefore, for the fixed CUDA version, we have fixed bandwidthTest version. Correct? What's the detailed version?

Reply:
cuda-samples is release together with CUDA, so yes, they have the exact same version. Like v10.0, v11.1 and so on.